src/file-manager/fm-tree-model.c

```diff
@@ -956,7 +957,9 @@ should_show_file (FMTreeModel *model, CajaFile *file)
 
     should = caja_file_should_show (file,
                                     model->details->show_hidden_files,
-                                    TRUE);
+				    /* TODO details->show_backup_files */
```

Comment:
This seems like a first attempt that got commented out. Please remove.

Reply:
Done (for the three ponts)